### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by DerekRoberts**
Basic README with architecture graphic, prerequisites, local setup and acknowledgements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+3. Start the Frontend Vue App (requires a new terminal session)
+
+   ```sh
+   cd frontend
+   npm install
+   npm run serve
+   ```
+
+4. Open the application. Frondend is the intended entry point.
```
Comment (BrownRhino) [inline README.md:127]: Change Frondend to Frontend

File: README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+A Common Services GETOK is available for free, approved use by the Government of British Columbia. Please see their project documentation.
```
Comment (BrownRhino) [inline README.md:73]: approved for use?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+```
+
+Install Node.js 10+ on Linux, OS X or Windows Subsystem for Linux v2 (WSL).
+
+- [Node.js Download](https://nodejs.org/en/download/)
+- [Windows Subsystem for Linux Installation (Windows Only)](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+### Installation - Local
+
+1. Clone, open and view the repo's contents, including hidden files
```
Comment (BrownRhino) [inline README.md:91]: Put in oxford comma so it's consistent with rest of the documentation (ex. Line 142):

Clone, open and view => Clone, open**,** and view

**Review by BrownRhino — DISMISSED** [review]
All good, just some typos that need to be addressed

File: README.md (hunk 1 above)
Comment (DerekRoberts) [reply README.md]: > All good, just some typos that need to be addressed

Thanks and fixed!  :))

**Review by BrownRhino — APPROVED** [review]
Good Work, Buddy!

**Pull request merged**